openlane/scripts/openroad/common/io.tcl

```diff
@@ -316,6 +316,35 @@ proc read_current_odb {args} {
     set_dont_use_cells
 }
 
+proc _populate_cells_by_class {} {
```

Comment:
I am wondering why this change at all.

Reply:
The idea is basically, those variables could be modified arbitrarily by the user. If a user chooses to customize these variables, the logical netlist may include some fill cells.